app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -23,7 +23,11 @@ import androidx.test.espresso.matcher.ViewMatchers.isRoot
 import androidx.test.espresso.matcher.ViewMatchers.withId
 import androidx.test.espresso.matcher.ViewMatchers.withText
 import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import com.google.common.truth.Truth.assertThat
 import dagger.Component
+import javax.inject.Inject
+import javax.inject.Singleton
```

Comment:
@rt4914 I am not able to help with this formatting as there is some issue with ktlint in my system, PTAL

Reply:
Both of these
```
import javax.inject.Inject
import javax.inject.Singleton
```

should come at the bottom of all the other imports.

Reply:
Done